demo_system/data_replay.py

```diff
@@ -158,14 +160,14 @@ def send_web_data(time, time_delta, is_abnormal):
 
 
 '''
-Send exception data start at 35,45,55 minutes in 4th hour.
+Send exception data start at 30,40,50 minutes in 4th hour.
 '''
 def send_log_data(time, time_delta, is_abnormal):
     timestamp = to_epochtime_minute(time)
     minute = get_time_delta_minute(time_delta)
     hour = get_time_delta_hour(time_delta)
     if is_abnormal:
-        if minute in [35,45,55] and hour == 0:
+        if minute in [30,40,50] and hour == 0:
```

Comment:
not sure if hour == 0 is the same logic as the 4th hour ?

Reply:
yep, this functionality has been testing on stg for several months.
If you are interested in the details can check the function **get_time_delta** and **get_time_delta_hour**
and generate a **config file** through the [tutorial](https://docs.google.com/document/d/1eBHKf4tOLgRcPIhekzBIMPnP-L9lDou0RxyTDQOdLAU/edit#heading=h.9iylwtjm1084) in Section 6.e